### DISCUSSION TITLE
RuntimeError: Creating a Parameter from an instance of type ParameterGGUF requires that detach() returns an instance of the same type, but return type Tensor was found instead.

### DISCUSSION THREAD

**Discussion by andy8992** [lllyasviel/stable-diffusion-webui-forge#1931] [Q&A]
I'm trying to use a GGUF and getting the above error. I have t5xxl_fp8_e4m3fn.safetensors, the other clip and the vae loaded. Isn't this all i need? 

Do i need an alternative clip file? can't find any info on this.

**Comment by seriousdev-gh** [comment]
Had the same error. It worked after replacing t5xxl_fp8_e4m3fn.safetensors with t5xxl_fp16.safetensors

**Reply by andy8992** [reply]
Thought that might be the case, bummer since it's almost 10gb, was hoping i'd save some space by getting GGuf models.

**Comment by ivans80** [comment]
> I'm trying to use a GGUF and getting the above error. I have t5xxl_fp8_e4m3fn.safetensors, the other clip and the vae loaded. Isn't this all i need?
> 
> Do i need an alternative clip file? can't find any info on this.

It is wrong. You have a wrong choice in "Diffusion in low bits" option. If you have wrong option there, you will get same error even if you change to t5xxl_fp16.safetensors. I change that to Automatic (didn't test all options), and you will solve error even if you have t5xxl_fp8_e4m3fn.safetensors loaded

But keep in mind, if model require fp16, maybe t5xxl_fp8_e4m3fn.safetensors will not work

![Capture](https://github.com/user-attachments/assets/73ab029d-6a84-4629-9463-06a8aaebf4da)
